### PR TITLE
Fixes #9209: Broken rpm agent builds in 4.0

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9209

### CODE REVIEW

**Comment by amousset** [comment]
WIP, do not merge for now

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
@Kegeruneku Ready for review!

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -166,6 +166,10 @@ ifneq (true,$(NO_LDSO))
 	$(INSTALL) -m 644 rudder.conf $(DESTDIR)/etc/ld.so.conf.d/
 endif
 
+        # Install /etc/bash_completion.d/rudder in order to autocomplete rudder commands
+	mkdir -p $(DESTDIR)/etc/bash_completion.d
+	$(INSTALL) -m 644 rudder.conf $(DESTDIR)/etc/bash_completion.d/
```
Comment (Kegeruneku) [inline rudder-agent/SOURCES/Makefile]: You sure this is the right file ??? rudder.conf is also installed in ld.so.conf...

**Comment by amousset** [comment]
Commit modified

File: rudder-agent/SOURCES/Makefile (hunk 1 above)
Comment (amousset) [inline rudder-agent/SOURCES/Makefile]: Indeed, the path was totally wrong...

**Comment by Kegeruneku** [comment]
Nice job

**Pull request merged**